src/dredd-command.coffee

```diff
@@ -219,9 +219,9 @@ class DreddCommand
       @serverProcess.on 'exit', =>
         logger.info('Backend server process exited')
 
-      @serverProcess.on 'error', (error) =>
-        logger.error('Command to start backend server process failed, exiting Dredd', error)
-        @_processExit(2)
+      @serverProcess.on 'spawnerror', (err) =>
```

Comment:
I have no problem with that, we can differentiate between different type of runtime errors in single error handler hub.